pkg/gadgets/trace/exec/tracer/bpf/execsnoop.bpf.c

```diff
@@ -84,6 +89,12 @@ int ig_execve_e(struct trace_event_raw_sys_enter* ctx)
 	event->args_size = 0;
 	event->mntns_id = mntns_id;
 
+	if (get_cwd) {
+		fs = BPF_CORE_READ(task, fs);
+		cwd = get_path_str(__builtin_preserve_access_index(&fs->pwd));
+		bpf_probe_read(event->cwd, MAX_STRING_SIZE, cwd);
```

Comment:
bpf_probe_read_kernel_str()
- Use the _kernel suffix since it is kernel memory
- Use _str for strings to avoid copying too much